src/hoi4_world/states/hoi4_states_converter.h

```diff
@@ -15,9 +15,13 @@
 namespace hoi4
 {
 
-States ConvertStates(const vic3::World source_world,
-    const mappers::WorldMapper world_mapper,
-    const hoi4::WorldFramework world_framework,
+std::map<std::string, int> MapVic3ProvincesToStates(const std::map<int, vic3::State>& states,
```

Comment:
With this now a public function, it should be directly tested. With luck, some of the existing tests might be adapted/replaced when doing so.